app/services/subscriptions/activate_service.rb

```diff
@@ -13,8 +13,10 @@ def activate_all_pending
         .joins(customer: :organization)
         .pending
         .where(previous_subscription: nil)
-        .where("DATE(subscriptions.subscription_at#{Utils::TimezoneService.at_time_zone}) <= "\
-               "DATE(?#{Utils::TimezoneService.at_time_zone})", Time.zone.at(timestamp))
+        .where(
+          "DATE(subscriptions.subscription_at#{Utils::Timezone.at_time_zone}) <= " \
+          "DATE(?#{Utils::Timezone.at_time_zone})", Time.zone.at(timestamp)
```

Comment:
Layout/MultilineMethodArgumentLineBreaks: Each argument in a multi-line method call must start on a separate line.